analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -124,6 +124,9 @@ def construct_analyzer_cmd(self, result_handler):
             # characters that would trigger globbing in the shell.
             analyzer_cmd.append("-checks=%s" % ','.join(checkers_cmdline))
 
+            # Display errors from all non-system headers.
+            analyzer_cmd.append("-header-filter=.*")
```

Comment:
This parameter should rather be passed as a default value of the
--analyzer-config because specifying it like this will be impossible to be overriden by the user.

See https://github.com/Ericsson/codechecker/pull/2685#pullrequestreview-398049157